integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/AwaitilityOnTimeOutLogger.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+import org.awaitility.core.ConditionEvaluationListener;
+import org.awaitility.core.EvaluatedCondition;
+import org.awaitility.core.TimeoutEvent;
+
+public class AwaitilityOnTimeOutLogger implements ConditionEvaluationListener {
+    private final Runnable runnable;
+
+    public AwaitilityOnTimeOutLogger(Runnable runnable) {
+        this.runnable = runnable;
+    }
+
+    @Override
+    public void conditionEvaluated(EvaluatedCondition condition) {
+        // nothing to do
+    }
+
+    @Override
+    public void onTimeout(TimeoutEvent timeoutEvent) {
+        runnable.run();
```

Comment:
As the class name is Awaitility**OnTimeOutLogger**, shouldn't we log something in `onTimeout` method?

My point is, I see that in new instance cration, we pass `runnable` that logs soemthing

```
new AwaitilityOnTimeOutLogger(() -> BridgeResource
                        .getBridgeDetailsResponse(context.getManagerToken(), bridgeContext.getId()).then().log().all()
```

however currently if developers pass `runnable` to `AwaitilityOnTimeOutLogger` that is sending some message to kafka, it is possible and can be confusing when reading the code. Or is it my lack of knowledge and the class name is appropriate?

Reply:
You are right, we should call it `AwaitilityOnTimeOutHandler`. wdyt ?

Reply:
@radtriste yes ^ that sounds better for me.

second option, if we always plan just log on timeout, that current class name is correct, just change constructor to take like `Supplier<String> logMessageSupplier`, just idea

Reply:
For now, we just want to log the rest failure when timeout. But this would be handled by the runnable we give and not by the `AwaitilityOnTimeOutLogger` itself. so I would rename it to `AwaitilityOnTimeOutHandler`

Reply:
renaming done